charts/atlas-operator-crds/Chart.yaml

```
@@ -2,8 +2,8 @@ apiVersion: v2
 name: mongodb-atlas-operator-crds
 description: MongoDB Atlas Operator CRDs - Helm chart for installing and upgrading Custom Resource Definitions (CRDs) for the Atlas Operator.
 type: application
-version: 1.9.1
-appVersion: 1.9.1
+version: 1.9.2
+appVersion: 1.9.2
```

Re-use good image 1.9.1 only the helm chart was wrong
```suggestion
appVersion: 1.9.1
```